deepmd/pt/model/model/make_hessian_model.py

```diff
@@ -185,7 +184,7 @@ def __init__(
             self,
             obj: CM,
             ci: int,
-            atype: torch.Tensor,
+            atype,
```

Comment:
why remove this

Reply:
For consistency with lines in `make_model.py`; now recovered.